lib/ansible/executor/powershell/module_manifest.py

```diff
@@ -294,6 +294,7 @@ def _create_powershell_wrapper(b_module_data, module_path, module_args,
         exec_manifest["actions"].insert(0, 'async_wrapper')
         exec_manifest["async_jid"] = str(random.randint(0, 999999999999))
         exec_manifest["async_timeout_sec"] = async_timeout
+        exec_manifest["win_async_startup_timeout"] = C.DEFAULT_WIN_ASYNC_STARTUP_TIMEOUT
```

Comment:
If you add a `vars` keyword to config, this needs to change to `C.config.get_config_value('WIN_ASYNC_STARTUP_TIMEOUT', variables=task_vars)` to allow it to come from the hostvars.